src/composables/useClipboard/useClipboard.test.ts

```diff
@@ -0,0 +1,10 @@
+import { describe, it, expect } from 'vitest';
+import useClipboard from '@/composables/useClipboard/useClipboard';
+
+describe('useClipboard Composable', () => {
+    it('clipboardText is empty on initialization', () => {
```

Comment:
why do we test for the clipboard being empty? Wouldn't it be more useful to check whether there is something in the clipboard after fireing "copyToClipboard"?